### DISCUSSION TITLE
Crossfeed (for headphones)?

### DISCUSSION THREAD

**Discussion by nji9nji9** [smplayer-dev/smplayer#1002] [Q&A]
Most of the time I take my headphones for a/v.
For that it is important to me to activate crossfeed (of the channels).
(Please don't mix up with "crossfade").
How do I do that with smplayer?
TIA

**Comment by Raza-0007** [comment]
In `Preferences -> Advanced -> Mplayer/mpv -> Options`

Type: `-af=crossfeed`

~Raza
